File: src/main/java/com/meetify/server/utils/StringUtils.java
package com.meetify.server.utils;

import java.util.Arrays;
import java.util.HashSet;
import java.util.List;

/**
 * com.meetify.server.utils
 * Created by kr3v on 08.10.2016.
 */
public class StringUtils {
    public static HashSet<Long> setFromString(String s) {
        HashSet<Long> answer = new HashSet<>();
        if (!s.equals(""))
            Arrays.asList(s.split(",")).forEach(s1 -> answer.add(Long.parseLong(s1)));
        return answer;
    }

    public static String makeString(List<String> request) {
        StringBuilder sb = new StringBuilder("");
        request.forEach(sb::append);
        return sb.toString();
    }
}

File: src/main/java/com/meetify/server/utils/StringUtils.kt
package com.meetify.server.utils

import java.util.*


object StringUtils {
    fun setFromString(s: String): HashSet<Long> {
        val answer = HashSet<Long>()
        if (s != "") {
            s.split(",").forEach { answer.add(it.toLong()) }
        }
        return answer
    }

    fun makeString(request: List<String>): String {
        val sb = StringBuilder("")
        request.forEach { sb.append(it) }
        return sb.toString()
    }
}
